daemon/algod/api/server/v2/handlers.go

```diff
@@ -732,6 +732,81 @@ func (v2 *Handlers) GetBlockTxids(ctx echo.Context, round uint64) error {
 	return ctx.JSON(http.StatusOK, response)
 }
 
+func NewBlockLog(txid string, logs []string, appIndex uint64) *model.BlockLog {
+	byteLogs := make([][]byte, len(logs))
+	for i, log := range logs {
+		byteLogs[i] = []byte(log)
+	}
+
+	return &model.BlockLog{
+		Txid:             txid,
+		Logs:             byteLogs,
+		ApplicationIndex: appIndex,
+	}
+}
```

Comment:
We have a `convertSlice` in this package you can use.
```suggestion
func NewBlockLog(txid string, logs []string, appIndex uint64) *model.BlockLog {
	return &model.BlockLog{
		Txid:             txid,
		Logs:             convertSlice(logs, fn (s string) { return []byte(s) })
		ApplicationIndex: appIndex,
	}
}
```